bin/kano-boot-splash-cli

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# kano-boot-splash-cli
+#
+# Set the boot splash for subsequent boots
+#
+
+"""
+Command line tool for changing settings on the Kano
+
+Usage:
+    kano-boot-splash-cli set <path>
+    kano-boot-splash-cli status
+    kano-boot-splash-cli clear
+    kano-boot-splash-cli -h | --help
+
+Commands:
+    status      Print the current boot splash path
+    set         Set the boot splash path
+    clear       Restore default splash
+"""
+
+import os
+import docopt
+import sys
+
+from kano.logging import logger
+
+RC_SUCCESS = 0
+RC_INCORRECT_ARGS = 1
+RC_INVALID_PATH = 2
+RC_WRONG_PERMISSIONS = 3
+
+SPLASH_LINK_PATH = '/var/cache/kano-desktop'
+SPLASH_LINK_FILENAME = 'boot_splash.link'
+
+
+def main(args):
+    linkpath = os.path.join(SPLASH_LINK_PATH, SPLASH_LINK_FILENAME)
+    if args['set']:
+        if args['<path>']:
```

Comment:
I think `docopt` will outright reject if you do not provide a value for `<path>` here, so you could remove the check.